plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -379,7 +379,8 @@ public class DeltaLakeMetadata
     public static final StorageFormat DELTA_STORAGE_FORMAT = create(
             LAZY_SIMPLE_SERDE_CLASS,
             SEQUENCEFILE_INPUT_FORMAT_CLASS,
-            HIVE_SEQUENCEFILE_OUTPUT_FORMAT_CLASS);
+            HIVE_SEQUENCEFILE_OUTPUT_FORMAT_CLASS,
+            ".seq");
```

Comment:
Do we need `.` as a prefix - why not as `seq`

Reply:
It's a file extension, so I think leading '.' is make sense.

Reply:
Do you think which class is most suitable for the adding

Reply:
added in the `BaseHiveConnectorTest`